### DISCUSSION TITLE
DTU W Lite S and OpenDTU parallel operation possible ?

### DISCUSSION THREAD

**Discussion by zigit780** [tbnobody/OpenDTU#1474] [Q&A]
Hello, I have a HMS-800-2T with an DTU W Iite S Stick from Hoymile and a OpenDTU with Oled-Display.
The openDTU use the CMT3200 rf-module.

Currently I wasn't sucessful to have parallel operation with both (OpenDTU and DTU Wlite S).
I changed the intervall of openDTU to 120s but the DTU-W lite S can't connect if openDTU is active.
and vice versa if DTU W lite S is active it seems I can't connect the openDTU.

So if I search with G***** I found statements that it should be possible in parallel  or is that a misunderstanding.
So must I increase intervall or must I set / not set something else in config for this.

Because I want to have the Live-Display and also the data evaluation in the cloud.

**Comment by stefan123t** [comment]
@zigit780 currently it is not intede to communicate with one inverter using two different DTUs simultaneously. This will lead to conflict and the very common error #2 DTU command failed.

There are intentions to build a DTU -> S-Miles Cloud bridge as the relevant .protobuf Network Command / S-Miles Installer App protocols are used to talk to newer WiFi series models.

Maybe these efforts will lead to theb option of  porting some of the .protobuf Protocol using the protobuf-nano Library to the OpenDTU ESP32 C-Code in the far future. But that is still debatable and has to be implemented / accepted by the project maintainer.

**Reply by zigit780** [reply]
But why I found some statements that it should be possible. I mean that it's not recommended is clear...

https://www.photovoltaikforum.com/thread/200323-hoymiles-dtu-wlite-vs-opendtu/?pageNo=2
"Also bei mir "meckert" keines der Geräte, habe DTU-Pro und Open DTU im Paralellbetrieb, funktioniert wunderbar."

Paralleloperation AoyDTU and OpenDTU
https://blog.helmutkarger.de/balkonkraftwerk-teil-8-opendtu-und-ahoydtu-fuer-hoymiles-wechselrichter/
"Danke Jan,
Deine Beobachtung kann ich zumindest für die Nachbau DTUs nicht bestätigen. Ich betreibe problemlos OpenDTU und Ahoy gleichzeitig auf einen Hoymiles Wechselrichter. Da kommt nichts durcheinander und selbstverständlich kann eine DTU die Einstellung einer anderen auf dem Wechselrichter (Leistungsbegrenzung) überschreiben."

And some others....

Or is it only working with the HM-Series with the other RF-modul ?

**Comment by dischij** [comment]
Hallo, bei mir ging es auch nicht.

https://github.com/tbnobody/OpenDTU/discussions/1386

Gruß
Jürgen

**Comment by turrican944** [comment]
Moin
Es geht bei der HM Serie recht problemlos mit zwei DTUs im parallelbetriebt. Bei den HMS/T geht es nicht, habe das paar mal probiert.

The HM series works quite easily with two DTUs. It doesn't work with the HMS/T, I tried that a few times.

**Comment by Tabaluga0815** [comment]
Look here, if you want to connect both:
https://github.com/tbnobody/OpenDTU/discussions/1218
